content/project/conduct.adoc

```diff
@@ -7,180 +7,252 @@ section: conduct
   link:https://fosdem.org[FOSDEM], link:https://socallinuxexpo.org/[SCaLE])
-* Other Jenkins groups (such as conferences, meetups, and other unofficial
-  forums) are encouraged to adopt this Code of Conduct. Those groups must
-  provide their own moderators and/or reporting system.
 
+=== Other Jenkins Communities
+
+Other Jenkins groups (such as conferences, meetups, and other unofficial forums) and local communities are encouraged to adopt this Code of Conduct.
+Those groups must provide their own moderators and/or reporting system.
 
 == Reporting
 
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the link:/project/board[Jenkins board].
 If you feel somebody has breached this code, please send an email with the
-relevant information (links, etc) to jenkinsci-board@googlegroups.com. This
-email list is only readable by the
-link:/project/board[Governance
-Board] members.
-
-Please understand that the board may not have all of the necessary context and
-history, so it's better to be more verbose than less verbose about such
-matters.
+relevant information (links, etc) to `jenkinsci-board@googlegroups.com`.
+This email list is only readable by the link:/project/board[Governance Board] members.
+The board may not have all of the necessary context and history,
+so it's better to be more verbose than less verbose about such matters.
+All complaints will be reviewed
+and investigated and will result in a response that is deemed necessary and
+appropriate to the circumstances.
 
 If you believe one of the members of the board has violated the code of conduct
 above, please email one of the other members of the Governance Board with the
 details (their emails are visible on the
 link:/project/board[Governance
 Board] page).
 
+If the desired resolution cannot be reached on the Jenkins community level,
+an issue can be escalated to the Continuous Delivery Foundation (CDF) by contacting the project team at `conduct@cd.foundation`.
+See the link:https://github.com/cdfoundation/toc/blob/master/CODE_OF_CONDUCT.md[CDF Code of Conduct] for more information about reporting and enforcement in this case.
+
 == Handling of violations
 
-Depending on the severity of the violations the board may elect to take one of the following paths. The resolution of violations will be done in private, and the affected people will be notified but there will not be a public announcement. We aim to prevent mud-slinging and unnecessary further personal attacks/etc.
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct.
 
-=== Reprimand
+Depending on the severity of the violations the board may elect to take one of the following paths.
+The resolution of violations will be done in private.
+And the affected people will be notified but there will not be a public announcement.
```

Comment:
```suggestion
The resolution of violations will be done in private and the affected people will be notified but there will not be a public announcement of the resolution.
```

Reply:
II am still not 100% sure about keeping it as is. We may be forced to publicly respond in some cases, so I would rather render it as a general and recommended practice, not a binding rule.

Reply:
That's fine, it's just generally not a good idea to start a sentence with "And" if its just a continuation of the previous sentence, so that was all my change was about.

Reply:
Right. Will move to a separate discussion